examples/unicoil/README.md

```diff
@@ -0,0 +1,20 @@
+## Train UniCOIL on MS MARCO
+```bash
+CUDA_VISIBLE_DEVICES=0 python examples/unicoil/train_unicoil.py \
```

Comment:
should we include the general coil setup and make unicoil as a specific setup flag

Reply:
yeah, let's do it in a separate PR?

Reply:
if we consider unicoil as a special case of coil, it may make sense to keep a single coil class for both. in that case, we probably needs to delay unicoil to the next pr. 
otherwise, we ship them as two separate classes, one in this one and one in the future pr.